Assets/HoloToolkit/Input/Scripts/GestureManager.cs

```diff
@@ -158,6 +175,25 @@ private void OnTap()
             }
         }
 
+        private void OnRecognitionStarted()
+        {
+            if (FocusedObject != null)
+            {
+                hasRecognitionStarted = true;
+                FocusedObject.SendMessage("OnPressed", SendMessageOptions.DontRequireReceiver);
```

Comment:
Hmm so you only need these for air tap? What about other gestures?
I believe the right place to add this might be in the HandsManager, where we detect FingerPressed and FingerReleased. They seem more generic. That will be agnostic to GR then.

What do you think?